tasks/21_check-letters/check-letters.js

```diff
@@ -1,6 +1,6 @@
 export function checkLetters(arr) {
-  let includedLetters = arr[1].toLowerCase().split('')
-      .map((a) => arr[0].toLowerCase().indexOf(a))
-      .filter((a) => a !== -1);
-  return includedLetters.length === arr[1].length;
+  arr = arr.map((element) => element.toLowerCase());
```

Comment:
[Reassignment of Function Parameters](https://eslint.org/docs/rules/no-param-reassign). It's better to create another variable